### PR TITLE
get answer

### PR DESCRIPTION
**Pull request by abdurezakarage**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -21,8 +23,12 @@ const questionsRoutes = require("./routes/questionRoutes.js");
 const authMiddleware = require("./middleware/authMiddleware");
 // questions routes middleware??
 app.use("/api/questions", authMiddleware, questionsRoutes);
+//answer route
+const answerroutes = require("./routes/getanswerroute.js");
```
Comment (mikiyasalehegn) [inline app.js]: Please use camel case!

File: app.js [hunk 2]
```diff
@@ -21,8 +23,12 @@ const questionsRoutes = require("./routes/questionRoutes.js");
...
 
-// answers routes middleware ??
+// answers routes middleware ?
+//answer route midilware
```
Comment (mikiyasalehegn) [inline app.js:30]: Remove this line!

File: app.js [hunk 3]
```diff
@@ -21,8 +23,12 @@ const questionsRoutes = require("./routes/questionRoutes.js");
 const authMiddleware = require("./middleware/authMiddleware");
 // questions routes middleware??
 app.use("/api/questions", authMiddleware, questionsRoutes);
+//answer route
+const answerroutes = require("./routes/getanswerroute.js");
 
-// answers routes middleware ??
+// answers routes middleware ?
```
Comment (mikiyasalehegn) [inline app.js]: Remove this line!

File: routes/getanswerroute.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const dbconnection = require("../db/dbconfig");
+const { StatusCodes } = require("http-status-codes");
+const getanswer = require("../controller/getanswercontroller");
+//
```
Comment (mikiyasalehegn) [inline routes/getanswerroute.js]: remove //

File: app.js [hunk 5]
```diff
@@ -21,8 +23,12 @@ const questionsRoutes = require("./routes/questionRoutes.js");
...
+//answer route midilware
+app.use("/api/answer", authMiddleware, answerroutes);
 async function start() {
   try {
     const result = await dbConnection.execute('select "test"');
```
Comment (mikiyasalehegn) [inline app.js]: no need of creating variable here.
just put this only -> ` await dbConnection.execute('select "test"');`

**Review by mikiyasalehegn — CHANGES_REQUESTED** [review]

File: app.js (hunk 5 above)
Comment (abdurezakham) [reply app.js]: Thanks miki. this one on the main branch from Gizish. I only add here answer endpoint api

File: app.js (hunk 3 above)
Comment (abdurezakham) [reply app.js]: ok

File: app.js (hunk 1 above)
Comment (mikiyasalehegn) [reply app.js:27]: done!

File: app.js (hunk 3 above)
Comment (abdurezakham) [reply app.js]: done

File: app.js (hunk 1 above)
Comment (abdurezakham) [reply app.js]: done

File: routes/getanswerroute.js (hunk 4 above)
Comment (abdurezakham) [reply routes/getanswerroute.js]: done

File: app.js (hunk 5 above)
Comment (abdurezakham) [reply app.js]: done

**Review by mikiyasalehegn — APPROVED** [review]
Well Done!

**Pull request merged**